src/tests/globalStyles.e2e.ts

```diff
@@ -32,19 +31,78 @@ describe("global styles", () => {
             opacity: opacity
           };
         });
-        expect(noticeAnimation.duration).toEqual("0.3s");
+        expect(noticeAnimation.duration).toEqual("0.15s");
         expect(noticeAnimation.name).toEqual(className.slice(className.indexOf("_") + 2));
         expect(noticeAnimation.opacity).not.toBe("0");
       });
     });
 
-    it("should have initial --calcite-animation-timing CSS Custom Property value", async () => {
-      const page = await newE2EPage({ html: snippet });
-      expect(
-        await page.evaluate(() => {
-          return window.getComputedStyle(document.body).getPropertyValue("--calcite-animation-timing");
-        })
-      ).toBe(" 300ms");
+    it("should set animation duration to 0ms", async () => {
```

Comment:
I'm not sure what this test is covering. `--calcite-animation-timing` is internal, so no need to test it, right? Maybe I'm missing something.

Reply:
This test was added prior to the audit. Might have overlooked and  this test makes more sense now considering `--calcite-animation-factor` will be the public.

Reply:
never mind. This test should be removed.